tools/flang2/flang2exe/llassem.cpp

```diff
@@ -1698,6 +1703,19 @@ write_comm(void)
 
     align_value = CACHE_ALIGN + 1;
 
+    /*
+      Create a tmp file and capture the initialization data.
+      Emitting the cmn blk var defn now will miss out the dbg info
+      generated for other references of this variable.
+    */
+    FILE *llvm_ir_bkup = gbl.asmfil;
+    FILE *fd;
```

Comment:
I suggest renaming these variables to clarify their purpose. `fd` is not a terribly useful name.

Reply:
Changed. Thanks.